kani-compiler/src/codegen_cprover_gotoc/utils/utils.rs

```diff
@@ -66,6 +67,20 @@ impl<'tcx> GotocCtx<'tcx> {
             _ => None,
         }
     }
+
+    /// Store an occurrence of a concurrent construct that was treated as a sequential operation.
+    ///
+    /// Kani does not currently support concurrency and the compiler assumes that when generating
+    /// code for some specialized concurrent constructs that this is the case. We store all types of
+    /// operations that had this special handling and print a warning at the end of the compilation.
+    pub fn store_concurrent_construct(&mut self, operation_name: &str, loc: Location) {
```

Comment:
As mentioned, printing the location isn't useful at the moment. A better thing to do would be storing locations from function calls up to here, then print whichever is closes to the user's code. But we don't have anything like that at the moment, right?